lib/components/admin/field-trip-details.js

```diff
@@ -0,0 +1,218 @@
+import { getDateFormat } from '@opentripplanner/core-utils/lib/time'
+import moment from 'moment'
+import React, { Component } from 'react'
+import { DropdownButton, MenuItem } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+
+import * as fieldTripActions from '../../actions/field-trip'
+import DraggableWindow from './draggable-window'
+import EditableSection from './editable-section'
+import FieldTripNotes from './field-trip-notes'
+import Icon from '../narrative/icon'
+import {
+  Bold,
+  Button,
+  Container,
+  FullWithMargin,
+  Half,
+  Header,
+  InlineHeader,
+  Para,
+  Text,
+  Val,
+  WindowHeader as DefaultWindowHeader
+} from './styled'
+import TripStatus from './trip-status'
+import Updatable from './updatable'
+import {
+  getGroupSize,
+  GROUP_FIELDS,
+  PAYMENT_FIELDS,
+  TICKET_TYPES
+} from '../../util/call-taker'
+
+const WindowHeader = styled(DefaultWindowHeader)`
+  margin-bottom: 0px;
+`
+
+/**
+ * Shows the details for the active Field Trip Request.
+ */
+class FieldTripDetails extends Component {
+  _editSubmitterNotes = (val) => this.props.editSubmitterNotes(this.props.request, val)
+
+  _getRequestLink = (path, isPublic = false) =>
+    `${this.props.datastoreUrl}/${isPublic ? 'public/' : ''}fieldtrip/${path}?requestId=${this.props.request.id}`
+
+  _onCloseActiveFieldTrip = () => this.props.setActiveFieldTrip(null)
+
+  _onClickCancel = () => {
+    const {request, setRequestStatus} = this.props
+    if (confirm('Are you sure you want to cancel this request? Any associated trips will be deleted.')) {
+      setRequestStatus(request, 'cancelled')
+    }
+  }
+
+  render () {
+    const {
+      addFieldTripNote,
+      dateFormat,
+      deleteFieldTripNote,
+      request,
+      setRequestGroupSize,
+      setRequestPaymentInfo,
+      style
+    } = this.props
+    if (!request) return null
+    const {
+      id,
+      invoiceRequired,
+      notes,
+      schoolName,
+      submitterNotes,
+      teacherName,
+      ticketType,
+      travelDate
+    } = request
+    const internalNotes = []
+    const operationalNotes = []
+    notes && notes.forEach(note => {
+      if (note.type === 'internal') internalNotes.push(note)
+      else operationalNotes.push(note)
+    })
+    const travelDateAsMoment = moment(travelDate)
+    return (
+      <DraggableWindow
+        footer={
```

Comment:
Can you move these footer and header items into their own `_renderHeader`, `_renderFooter` methods so that this render function is better organized and not so big?